test/integration/hooks/validate.test.js

```diff
@@ -41,6 +41,74 @@ describe(Support.getTestDialectTeaser('Hooks'), function() {
       });
     });
 
+    describe('#3534, hooks modifications', function() {
+      it('fields modified in hooks are saved', function() {
```

Comment:
actually when test will run it will show like 

```
  [MYSQL] Hooks
      #validate
         #create
            ✓ should return the user
         #3534, hooks modifications
           ✓ fields modified in hooks are saved
```

Thats why I didn't included validation in description